### PR TITLE
Update all non-crate deps

### PR DESCRIPTION
**Pull request by grod220**
A follow up to https://github.com/penumbra-zone/web/pull/839 in preparation for release

### CODE REVIEW

File: apps/extension/src/content-scripts/injected-penumbra-global.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ const postRequest = () => {
       window.origin,
     );
     request.promise
-      .catch(e => connection.reject(e))
+      .catch((e: unknown) => connection.reject(e))
```
Comment (grod220) [inline apps/extension/src/content-scripts/injected-penumbra-global.ts:90]: New eslint rule: https://typescript-eslint.io/rules/use-unknown-in-catch-callback-variable/

Catch needs to specify that it's an unknown

File: packages/ui/package.json [hunk 2]
```diff
@@ -33,37 +33,37 @@
...
-    "framer-motion": "^11.0.8",
+    "framer-motion": "^11.0.22",
     "humanize-duration": "^3.31.0",
-    "lucide-react": "^0.354.0",
+    "lucide-react": "^0.363.0",
     "react-dom": "^18.2.0",
     "react-json-view-lite": "^1.3.0",
     "react-loader-spinner": "^6.1.6",
     "react-router-dom": "^6.22.3",
     "sonner": "1.4.3",
```
Comment (grod220) [inline packages/ui/package.json:43]: Sonner has an update, but the latest version has a bug: https://github.com/emilkowalski/sonner/issues/378. We need to wait a bit one this one.

File: packages/client/src/prax.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ import { createChannelTransport } from '@penumbra-zone/transport-dom/src/create'
 import { PenumbraSymbol } from './global';
 import { jsonOptions } from '@penumbra-zone/types/src/registry';
 
-const prax_id = 'lkpmkhpnhknhmibgnmmhdhgdilepfghe' as const;
+const prax_id = 'lkpmkhpnhknhmibgnmmhdhgdilepfghe';
```
Comment (grod220) [inline packages/client/src/prax.ts:14]: Eslint rule that is was an unnecessary const

File: packages/eslint-config-custom/index.js [hunk 4]
```diff
@@ -59,6 +59,7 @@ module.exports = {
         ],
       },
     ],
+    '@typescript-eslint/restrict-template-expressions': ['error', { allowNumber: true }],
```
Comment (grod220) [inline packages/eslint-config-custom/index.js:62]: New eslint rule: https://typescript-eslint.io/rules/restrict-template-expressions/

I think this makes sense for everything except for numbers. I think we all know that putting a number into a string template stringifies it and there isn't any crazy things like `[object Object]` to worry about.

File: turbo.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://turbo.build/schema.json",
   "globalDependencies": ["**/.env.*local"],
-  "globalEnv": ["NODE_ENV"],
+  "globalEnv": ["NODE_ENV", "MODE"],
```
Comment (grod220) [inline turbo.json:4]: New turbo eslint rule that requires us to put env variables here

**Comment by TalDerei** [comment]
LGTM

**Review by TalDerei — APPROVED** [review]

**Pull request merged**